_component_design/assisted-search.md

```diff
@@ -8,7 +8,7 @@ usage: >
 preview-image: preview-images/assisted-search.svg
 resource: true
 status: stable
-last-modified: 2017-11-07
+last-modified: 2018-02-12
```

Comment:
?

Reply:
Ugh, yes I edited that PR just prior to this one, must have edited the date in the wrong file. Good news is this is technically accurate, should I change this back or leave it as is?